flink-python/pyflink/table/expression.py

```diff
@@ -1354,6 +1370,27 @@ def sha2(self, hash_length: Union[int, 'Expression[int]']) -> 'Expression[str]':
         """
         return _binary_op("sha2")(self, hash_length)
 
+    # ---------------------------- JSON functions -----------------------------
+
+    def json_exists(self, path: str, on_error: JsonExistsOnError = None) -> 'Expression[bool]':
+        """
+        Determines whether a JSON string satisfies a given search criterion.
+
+        This follows the ISO/IEC TR 19075-6 specification for JSON support in SQL.
+
+        Example:
+        ::
+
+            >>> col('f0').json_exists('lax $.a.b[0].c')
+
+        Known Limitations: Lax mode currently does not unnest arrays automatically. This is an
```

Comment:
nit: I think we can remove the limitations from the JavaDocs and only mention them in the website docs